spec/lib/identity_tijuana_pull_spec.rb

```diff
@@ -82,6 +82,16 @@ def phone_numbers_are_equivalent(phone1, phone2)
         expect(u.postcode.number).to eq(m.address.postcode)
         expect(u.postcode.state).to eq(m.address.state)
       end
+      it 'creates new users in Tijuana without postcode' do
```

Comment:
@michael-gratton not sure if this is what you meant to test sync with invalid postcode?

Reply:
Ah no, so a user with a postcode that cannot exist.

Something like:

```
m = FactoryBot.create(:member)
m.addresses.create!({postcode: "invalid"})
```